app/test_main.py

```diff
@@ -1,3 +1,79 @@
+from typing import Any, List
+
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected",
+        [
+            (0, 0, [0, 0]),
```

Comment:
Add test cases with negative numbers and numbers that are different for `cat_age` and `dog_age` as mentioned in the checklist